documentation/experimental_features.md

```diff
@@ -327,3 +327,46 @@ To encrypt SSH connections using the unsupported algorithm
 > option to shell out to SSH. See [the net-ssh
 > README](https://github.com/net-ssh/net-ssh/#supported-algorithms) for a list of supported
 > algorithms.
+
+## `bolt plan new` command
+
+This feature was introduced in [Bolt
+2.22.0](https://github.com/puppetlabs/bolt/blob/main/CHANGELOG.md#bolt-2220-2020-08-10).
+
+The `bolt plan new` command is used to generate a new project-level YAML plan.
+It accepts a single argument: the name of the plan. To use the `bolt plan new`
```

Comment:
```suggestion
The command accepts a single argument: the name of the plan. To use the `bolt plan new`
```
For a second I parsed this as "The plan that is generated accepts the name of the plan"

Reply:
Though definitely feel free to ignore - I'm hopefully the only one who would read it as such